pkgs/development/python-modules/scikit-fmm/default.nix

```diff
@@ -1,22 +1,52 @@
 {
   lib,
   buildPythonPackage,
+  fetchpatch,
   fetchPypi,
+  meson-python,
   numpy,
   python,
 }:
 
 buildPythonPackage rec {
   pname = "scikit-fmm";
   version = "2023.4.2";
-  format = "setuptools";
+  pyproject = true;
 
   src = fetchPypi {
     inherit pname version;
     hash = "sha256-14ccR/ggdyq6kvJWUe8U5NJ96M45PArjwCqzxuJCPAs=";
   };
 
-  propagatedBuildInputs = [ numpy ];
+  # TODO: Remove these patches after another stable release is made.
+  # For now, these allow us to build with Python 3.12+ by switching to Meson
+  # and off the deprecated distutils.
+  patches = [
+    (fetchpatch {
+      name = "first try at Meson build";
```

Comment:
Not sure if we should replace the spaces by dashes or if that is done automatically

Reply:
Yeah ideally this would be `first-try-at-meson-build.patch`

Reply:
Resolved in the latest -- I changed them all to be lowercase and use dashes.